collections/_pages/features.html

```diff
@@ -0,0 +1,100 @@
+---
+layout: page
+permalink: /features
+title: Features
+heading: Meshery is <i>the</i> service mesh management plane
```

Comment:
Heading is fine, 
![A1](https://user-images.githubusercontent.com/48355572/89727308-ee5c2600-da40-11ea-813a-53940b99f403.JPG)

But maybe if we remove the italics from "the", it might look better.
![A2](https://user-images.githubusercontent.com/48355572/89727319-13e92f80-da41-11ea-9c0b-22e240488551.JPG)

Not a matter of concern though :smile: 
Agree if you feel the same :)